tests/redirects_tests/tests.py

```diff
@@ -86,3 +86,15 @@ def test_response_redirect_class(self):
         Redirect.objects.create(site=self.site, old_path='/initial/', new_path='/new_target/')
         response = self.client.get('/initial/')
         self.assertEqual(response.status_code, 302)
+
+
+class DisallowedRedirectTests(TestCase):
```

Comment:
The `redirects_tests` package is for `contrib.redirects`. `httpwrappers` looks like a better place for the test.